src/libYARP_dev/src/modules/RemoteControlBoard/RemoteControlBoard.cpp

```diff
@@ -961,6 +962,29 @@ class yarp::dev::RemoteControlBoard :
         remote = config.find("remote").asString().c_str();
         local = config.find("local").asString().c_str();
 
+        // check the Qos perefernces if available (local and remote)
+        yarp::os::QosStyle localQos;
+        if (config.check("local_qos")) {
+            Bottle& qos = config.findGroup("local_qos");
+            if(qos.check("thread_priority"))
+                localQos.threadPriority = qos.find("thread_priority").asInt();
+            if(qos.check("thread_policy"))
+                localQos.threadPolicy = qos.find("thread_policy").asInt();
```

Comment:
I'd check here if the input is actually an integer or not. Unexperienced user may type things like "FIFO" as policy.
Do values have Min/Max ranges? Could worth check that too.